modules/aruco/samples/calibrate_camera_charuco.cpp

```diff
@@ -57,8 +57,8 @@ const char* about =
 const char* keys  =
         "{w        |       | Number of squares in X direction }"
         "{h        |       | Number of squares in Y direction }"
-        "{sl       |       | Square side lenght (in pixels) }"
-        "{ml       |       | Marker side lenght (in pixels) }"
+        "{sl       |       | Square side length (in meters) }"
+        "{ml       |       | Marker side length (in meters) }"
```

Comment:
Ow I actually mixed things up, you are correct, this should indeed be units of length. But is meters the right parameter then? Shouldnt we use cm or mm?